cluster/gce/util.sh

```diff
@@ -527,10 +527,16 @@ function create-node-template() {
     "${IP_ALIAS_SUBNETWORK:-}" \
     "${IP_ALIAS_SIZE:-}")
 
+  local accelerator_args=""
+  # VMs with Accelerators cannot be live migrated.
+  # More details here - https://cloud.google.com/compute/docs/gpus/add-gpus#create-new-gpu-instance
+  if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
+      accelerator_args="--maintenance-policy TERMINATE --restart-on-failure --accelerator ${NODE_ACCELERATORS}"
```

Comment:
does it matter that terminate is all caps?  Looks like [the documentation](https://cloud.google.com/compute/docs/instances/setting-instance-scheduling-options#setnewoption) has it lowercase...